integration_tests/src/main/python/hive_write_test.py

```diff
@@ -51,13 +51,13 @@ def _restricted_timestamp(nullable=True):
     lambda nullable=True: DecimalGen(precision=15, scale=1, nullable=nullable),
     lambda nullable=True: DecimalGen(precision=36, scale=5, nullable=nullable)]]
 
-_write_gens = [_basic_gens, _struct_gens, _array_gens, _map_gens]
+_write_gens = [_basic_gens]  # , _struct_gens, _array_gens, _map_gens]
 
 # There appears to be a race when computing tasks for writing, order can be different even on CPU
 @ignore_order(local=True)
 @pytest.mark.skipif(not is_hive_available(), reason="Hive is missing")
 @pytest.mark.parametrize("gens", _write_gens, ids=idfn)
-@pytest.mark.parametrize("storage", ["PARQUET", "nativeorc", "hiveorc"])
+@pytest.mark.parametrize("storage", ["PARQUET"])  # , "nativeorc", "hiveorc"])
```

Comment:
Looks like leftovers from debugging.

Reply:
Sorry, thank for pointing that out. Fixed now.